buildSrc/src/main/java/org/opensearch/gradle/DistributionDownloadPlugin.java

```diff
@@ -73,6 +73,8 @@ public class DistributionDownloadPlugin implements Plugin<Project> {
     private static final String RELEASE_PATTERN_LAYOUT = "/core/opensearch/[revision]/[module]-min-[revision](-[classifier]).[ext]";
     private static final String SNAPSHOT_PATTERN_LAYOUT =
         "/snapshots/core/opensearch/[revision]/[module]-min-[revision](-[classifier])-latest.[ext]";
+    private static final String BUNDLE_PATTERN_LAYOUT =
+        "/distribution-build-opensearch/[revision]/latest/linux/x64/tar/dist/opensearch/[module]-[revision](-[classifier]).[ext]";
```

Comment:
Need to check how to use dynamic arch/platform/distro in the string pattern, instead of hardcoding x64/linux/tar.

Reply:
@peterzhuamazon This is probably not applicable as I did quite some research and didn't find a way to modify these pattern placeholder. The `[classifier]` in this case is `linux-x64` but I'm not pretty aware how we could change to `linux/x64` it as it's executed by gradle plugin. Open to any suggestions.

Also I see we have quite some bwcTests related are hardcoded to linux x64 and so were the BWC tests actively running in component repo are all hardcoded to use x64 architecture.

Reply:
@zelinh trying to understand whats the contention here? Is it with the format of `linux-x64` vs `linux/x64`?
This won't work for other distributions (windows, linux arm64 etc)

Reply:
@saratvemulapalli Yes right. The current `classifier` is `linux-x64` and can't be used for the path as `linux/x64` here. The reason I hardcoded here is I see it's also hardcoded and used in component bwc plugin tests for example https://github.com/opensearch-project/anomaly-detection/blob/bae45dbb924d3b04e2838d9e8b1b62de95467ac0/build.gradle#L41
So we think we could kick off and give a try with x64 linux first.

Reply:
makes sense. We dont want to block this PR, but could you open an issue. 
This should keep you going for the first phase

Reply:
@saratvemulapalli Thanks. I raised an issue regarding of this discussion here. https://github.com/opensearch-project/OpenSearch/issues/5535